plugins/knife/clearwater-security-groups.rb

```diff
@@ -280,7 +280,14 @@ def cw_aio_security_group_rules
   ]
 end
 
-def clearwater_security_groups
+def reformat_custom_groups(map)
+  (map.map do |group, ports|
+    [{ ip_protocol: :tcp, min: ports[0], max: ports[1], group: group },
+    { ip_protocol: :udp, min: ports[0], max: ports[1], group: group }]
```

Comment:
Why do we expose TCP and UDP every time (I think this is related to my later comment about generality)?